main/src/main/java/cgeo/geocaching/ImageEditActivity.java

```diff
@@ -140,6 +141,21 @@ public void onCreate(final Bundle savedInstanceState) {
         }
 
         loadImage();
+
+        // check the image details
+        binding.logImgViewBtn.setOnClickListener(view -> {
```

Comment:
Why have an extra button and not open the image viewer when clicking on the image itself?